interpreter/cling/lib/Interpreter/Exception.cpp

```diff
@@ -13,8 +13,18 @@
 #include "clang/Basic/SourceLocation.h"
 #include "clang/Sema/Sema.h"
 #include "clang/Sema/SemaDiagnostic.h"
+#include "clang/AST/ASTContext.h"
+
 
 extern "C" {
+void cling__runtime__internal__check__and__throw__if__invalid(void* Sema, void* Expr, const void* Arg) {
+  if (Arg)
+    return;
+  clang::Sema* S = (clang::Sema*)Sema;
+  clang::Expr* E = (clang::Expr*)Expr;
+  throw cling::NullDerefException(S, E);
+}
+
 void cling__runtime__internal__throwNullDerefException(void* Sema, void* Expr) {
```

Comment:
Not really, I grep-ed for it and removed it from RequiredSymbols, that the only place I found it.